From 657819ff477dd73cd71075609698aa57ba098d8c Mon Sep 17 00:00:00 2001
From: David Abdurachmanov <david.abdurachmanov@sifive.com>
Date: Wed, 15 Sep 2021 07:10:02 -0700
Subject: [PATCH 2/7] riscv: sifive: unmatched: update regulators values

These are the regulators values from the schematics for Rev3{A,B} boards.

Note this is not fully correct as bcore1/bcore2 and bmem/bio are merged, but
it's only supported in v5.15 kernel. See:

541ee8f640327f951e7039278057827322231ab0 ("regulator: da9063: Add support for
full-current mode.")

This will be changed for v5.15 kernel based on the patch above.

Signed-off-by: David Abdurachmanov <david.abdurachmanov@sifive.com>
---
 .../riscv/boot/dts/sifive/hifive-unmatched-a00.dts | 32 +++++++++++-----------
 1 file changed, 16 insertions(+), 16 deletions(-)

--- a/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts
+++ b/arch/riscv/boot/dts/sifive/hifive-unmatched-a00.dts
@@ -74,16 +74,16 @@
 
 		regulators {
 			vdd_bcore1: bcore1 {
-				regulator-min-microvolt = <900000>;
-				regulator-max-microvolt = <900000>;
+				regulator-min-microvolt = <1050000>;
+				regulator-max-microvolt = <1050000>;
 				regulator-min-microamp = <5000000>;
 				regulator-max-microamp = <5000000>;
 				regulator-always-on;
 			};
 
 			vdd_bcore2: bcore2 {
-				regulator-min-microvolt = <900000>;
-				regulator-max-microvolt = <900000>;
+				regulator-min-microvolt = <1050000>;
+				regulator-max-microvolt = <1050000>;
 				regulator-min-microamp = <5000000>;
 				regulator-max-microamp = <5000000>;
 				regulator-always-on;
@@ -138,48 +138,48 @@
 			};
 
 			vdd_ldo3: ldo3 {
-				regulator-min-microvolt = <1800000>;
-				regulator-max-microvolt = <1800000>;
+				regulator-min-microvolt = <3300000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-min-microamp = <200000>;
 				regulator-max-microamp = <200000>;
 				regulator-always-on;
 			};
 
 			vdd_ldo4: ldo4 {
-				regulator-min-microvolt = <1800000>;
-				regulator-max-microvolt = <1800000>;
+				regulator-min-microvolt = <2500000>;
+				regulator-max-microvolt = <2500000>;
 				regulator-min-microamp = <200000>;
 				regulator-max-microamp = <200000>;
 				regulator-always-on;
 			};
 
 			vdd_ldo5: ldo5 {
-				regulator-min-microvolt = <1800000>;
-				regulator-max-microvolt = <1800000>;
+				regulator-min-microvolt = <3300000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-min-microamp = <100000>;
 				regulator-max-microamp = <100000>;
 				regulator-always-on;
 			};
 
 			vdd_ldo6: ldo6 {
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
+				regulator-min-microvolt = <1800000>;
+				regulator-max-microvolt = <1800000>;
 				regulator-min-microamp = <200000>;
 				regulator-max-microamp = <200000>;
 				regulator-always-on;
 			};
 
 			vdd_ldo7: ldo7 {
-				regulator-min-microvolt = <1800000>;
-				regulator-max-microvolt = <1800000>;
+				regulator-min-microvolt = <3300000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-min-microamp = <200000>;
 				regulator-max-microamp = <200000>;
 				regulator-always-on;
 			};
 
 			vdd_ldo8: ldo8 {
-				regulator-min-microvolt = <1800000>;
-				regulator-max-microvolt = <1800000>;
+				regulator-min-microvolt = <3300000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-min-microamp = <200000>;
 				regulator-max-microamp = <200000>;
 				regulator-always-on;
